cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -95,18 +95,15 @@ func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadm
 	etcdPeerAddress := etcdutil.GetPeerURL(&cfg.LocalAPIEndpoint)
 
 	klog.V(2).Infof("[etcd] get the member id from peer: %s", etcdPeerAddress)
-	id, err := etcdClient.GetMemberID(etcdPeerAddress)
-	if err != nil {
-		return err
-	}
-
-	klog.V(1).Infof("[etcd] removing etcd member: %s, id: %d", etcdPeerAddress, id)
-	members, err := etcdClient.RemoveMember(id)
-	if err != nil {
+	if id, err := etcdClient.GetMemberID(etcdPeerAddress); err != nil {
 		return err
+	} else {
```

Comment:
this change will probably not pass golint [1]
please take a look at the CI report.